apps/web/vibes/index.ts

```diff
@@ -1,3 +1,3 @@
-export { default as eclipse } from './eclipse'
+// export { default as eclipse } from './eclipse'
```

Comment:
Should we go ahead and delete these commented out lines? The other branches should have them anyway so we don't need to keep them around.